src/client/pages/Landing/landingPageData.ts

```diff
@@ -0,0 +1,87 @@
+import { Market } from 'components/utils/CurrentLocale'
+
+type Locale = 'se' | 'se-en' | 'no' | 'no-en' | 'dk' | 'dk-en'
```

Comment:
Should we name this something other than `Locale` to not mix it up with the real Locale we get from the schema? Something like `LocalePath` to highlight what it's used for 🤷‍♂️

Reply:
Hm, yes probably - good point! Since this is only used locally (hehe) in this file maybe change this later on when we will probably use this globally?